core/less/layout/menu/menu.less

```diff
@@ -25,16 +26,16 @@
             min-width: 115px;
             display: inline-block;
             float: none;
+            color: white;
 
             a {
+                opacity: 0.9;
+                color: white !important;
```

Comment:
Yes I know. But this file is yet to be refactor in a cleaner way. I just changed it to improve things a little regarding the navbar. The important here is not really a issue, given the high specificity of the selector before.